### PR TITLE
add instructions for fixing broken gitlab authentication

### PR DESCRIPTION
**Pull request by ateucher**

### CODE REVIEW

**Review by BonnieJRobert — APPROVED** [review]

File: docs/new-sre-machine-setup.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
...
-   - [ ] If an existing project (but a new machine), type `cd dipr && git pull`. This will prompt an authentication window where you type in your username and your PAT from step 2 as your password (if this is a new machine for an existing project, you should have your PAT in the text file you previously saved in step 2). Your gitlab credentials will now be stored for all git projects on this machine.
+   - [ ] If an existing project (but a new machine), type `cd dipr && git pull`. This will prompt an authentication window where you type in your username and your PAT from step 2 as your password (if this is a new machine for an existing project, you should have your PAT in the text file you previously saved in step 2). Your gitlab credentials will now be stored for all git projects on this machine. Note: if you entered your PAT incorrectly the incorrect credentials will be stored and you will be unable to interact with gitlab. You can change/reset it by opening Credential Manager in the Windows Control Panel and under "Windows Credentials" there will be a gitlab entry you can then edit.
```
Comment (boshek) [inline docs/new-sre-machine-setup.md:15]: I wonder `dipr::set_gitlab_credentials` could also work here.

Comment (ateucher) [reply docs/new-sre-machine-setup.md:15]: Oh I didn't know that was a thing! 🤦

Comment (ateucher) [reply docs/new-sre-machine-setup.md:15]: However there is a good chance that if you run into this problem you haven't yet installed dipr...

Comment (boshek) [reply docs/new-sre-machine-setup.md:15]: Great point

**Review by boshek — APPROVED** [review]

**Pull request merged**